include/bluetooth/services/bas_c.rst

```diff
@@ -32,6 +32,10 @@ Reading the current value
   Use :cpp:func:`bt_gatt_bas_c_read` to read the battery level.
   When subscribing to notifications, you can call this function to retrieve the current value even when there is no change.
 
+Reading the current value with time interval
+  Use :cpp:func:`bt_gatt_bas_c_read_interval_set` to automatically read the battery level with given time interval.
```

Comment:
to periodically read the ...
For consistency is description with header file.

Reply:
Ok